### DISCUSSION TITLE
Add "Copy/Edit Raw Value" to Field Label Menu

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3098] [Feature Requests]
We already have a menu for form field labels, I propose we add another option here that allows users to copy the field's raw value to their clipboard. Perhaps even lets users **edit** the raw value directly (in a modal)... which might make copying/pasting data easier for some interfaces (eg: non-relational repeaters).

**Comment by connorwinston** [comment]
This was implemented.
